modules/viz/misc/python/test/test_simple.py

```diff
@@ -0,0 +1,440 @@
+import os
```

Comment:
> modules/viz/misc/python/test/test_simple.py

Please rename to `test_viz_simple.py` (similar to python tests of other modules).